extensions/quick-open-project/package.json

```diff
@@ -42,6 +43,14 @@
           "description": "The editor app to use when opening a project folder",
           "default": "/Applications/Visual Studio Code.app"
         },
+        {
+          "name": "editorAppAlt",
+          "type": "appPicker",
+          "required": false,
+          "title": "Alternate Editor",
+          "description": "The alternate editor app to use when opening a project folder",
+          "default": "/Applications/PhpStorm.app"
```

Comment:
In a similar line of thinking, my contribution added the ability to include .vscode workspace files, which has been tremendously useful for me since I use that feature. But anyone not using VSCode as their main (or secondary) editor might actually want the equivalent of their own editor's workspaces files showing up in the results. Maybe we/I should rework that feature into something where the `Paths` glob looks at filename patterns as well as just folders?

Reply:
@bjw-s That seems well beyond my intent with this PR, but I could see how either adjusting or exposing the glob might be useful!

Reply:
> @bjw-s That seems well beyond my intent with this PR, but I could see how either adjusting or exposing the glob might be useful!

Oh for sure it shouldn't be implemented in this PR, but I figured this could be a good place to discuss the desirability of the  change(s) 😅